target/linux/ramips/dts/KE2P.dts

```diff
@@ -0,0 +1,127 @@
+/dts-v1/;
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "phicomm,ke2p", "mediatek,mt7621-soc";
+	model = "Phicomm KE 2P";
```

Comment:
Please decide on which is the correct version, "Phicomm KE2P" or "Phicomm KE 2P", and then use that one consistently.
If the correct name is "Phicomm KE 2P", the compatible would be "phicomm,ke-2p" and the device name in Makefile "phicomm_ke-2p".

Reply:
Understood. It's sold and labeled as "KE 2P", so let's go for ke-2p.